MBBSEmu.Tests/ExportedModules/Majorbbs/setmem_Tests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
```

Comment:
could you also update the implementation of `setmem`?

We should be able to do `Module.Memory.FillArray` instead of the loop that it currently executes

Reply:
Updated setmem() and memset() to use FillArray, added debug to setmem()